inference-engine/src/transformations/include/ngraph_ops/swish_ie.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph {
+namespace op {
+class TRANSFORMATIONS_API SwishIE : public Op {
+public:
+    static constexpr NodeTypeInfo type_info{"SwishIE", 1};
+    const NodeTypeInfo &get_type_info() const override { return type_info; }
+
+    explicit SwishIE(const Output<Node> &input, float alpha = 1.0);
+
+    void validate_and_infer_types() override;
+    bool visit_attributes(AttributeVisitor& visitor) override;
+    std::shared_ptr<Node> clone_with_new_inputs(const OutputVector &new_args) const override;
+
+    float m_alpha;
```

Comment:
Why do we have class member in public?

Reply:
Fixed. Now the attribute is protected and I have created setter and getter.

Reply:
Why protected? Do we have a plan use it in other nodes?

Reply:
May be in the future we will need another version of the operation so it will be easier to make an attribute protected. Do you have strong rationale to make in private?

Reply:
No, I just wanted to understand the reason :)